### DISCUSSION TITLE
Adding keypad Buzzer output on ESP8266

### DISCUSSION THREAD

**Discussion by Welpie22** [taligentx/dscKeybusInterface#251] [dscKeybusInterface Discussion]
Hi guys

I'm new to this whole scene and it seems I have burnt out my keypad somehow with the setup.
I'm using the Virtual Keypad version and would like to know if a buzzer can be connected to the ESP8266 to buzz like the original keypad did?

**Comment by kirandanuwar** [comment]
i have nodemcu esp8266. i am trying make a perfect wifi hacking machine with it.. which can perform wireshark, evil twin, aircrack, handshake capture and web interface to access it. but it's hard, alot of coding is needed. i want to work in team. if agree, please reply
